core/domain/user_services_test.py

```diff
@@ -335,7 +409,51 @@ def test_get_user_settings_by_auth_id_strict_returns_user_settings(self):
     def test_get_user_settings_by_auth_id_strict_for_missing_auth_id_is_none(
             self):
         with self.assertRaisesRegex(Exception, 'User not found.'):
-            user_services.get_user_settings_by_auth_id('auth_id_x', strict=True)
+            user_services.get_user_settings_by_auth_id(
+                'auth_id_x',
+                strict=True
+        )
+
+    def test_get_users_setting_system_user_id_adds_system_user_setting(self):
```

Comment:
This doesn't really scan; can you try rewording it?

Reply:
I couldn't really figure out a better way to word this without going over 80 cols, am I allowed to abbreviate some words (e.g. system --> sys) so that it still passes the linting checks?

Reply:
Something like test_get_users_setting_retrieves_settings_for_system_user(), perhaps?